app/assets/stylesheets/application.scss

```diff
@@ -43,6 +64,14 @@
 
  }
 
+ .post-it-placeholder {
+ background: $dark-grey;
+ padding:15px;
```

Comment:
Properties should be ordered background, font-size, padding, width<br>Colon after property should be followed by one space